Conference-Room-Reservation-System/Conference-Room-Reservation-System/src/main/java/com/sda/conferenceroomreservationsystem/service/ReservationService.java

```diff
@@ -20,31 +22,30 @@ public class ReservationService {
     private final ConferenceRoomRepository conferenceRoomRepository;
     private final OrganizationRepository organizationRepository;
     private final ReservationRepository reservationRepository;
+    private final ReservationMapper reservationMapper;
 
-    public List<ReservationDto> getAllReservations(){
-        final List<Reservation> reservations = reservationRepository.findAll();
-        return null;
+    public List<ReservationDto> getAll(){
+        return reservationRepository.findAll().stream().map(ReservationMapper::map).collect(Collectors.toList());
```

Comment:
I would simplify to "return reservationRepository.findAll().stream().map(ReservationMapper::map).toList();"